app/styles/nprogress.css

```diff
@@ -0,0 +1,13 @@
+#nprogress .bar {
+  background: linear-gradient(
+    to right,
+    #ef6820,
+    #ef6820
+  );
+}
+
+#nprogress .peg {
+  box-shadow:
+    0 0 10px #ef6820,
```

Comment:
Please use apply and the color classes from tailwind. This needs to be dynamic as if its not, when we decide to change the main color, it will be a nightmare, and we already have enterprise clients that want that. You can just use @apply from tailwind to give it the correct color. It should be `primary-700`